addons/languages/oldnortharabian/apk/build.gradle

```diff
@@ -0,0 +1,13 @@
+ext.shouldBePublished = true
```

Comment:
Since this is a new pack (that has not been published yet), you need to set it `false`. So:
```
ext.shouldBePublished = false
ext.notPublishingReason = "Initial version has not been published yet"
```